app/presenters/content_item_presenter.rb

```diff
@@ -44,14 +44,28 @@ def as_json(options = nil)
 
   def links
     return item.expanded_links unless document_type_expanded?
-    Rails.application.statsd.time('public_content_item_presenter.links') do
-      item.linked_items.each_with_object({}) do |(link_type, linked_items), items|
-        items[link_type] = linked_items.map { |i| present_linked_item(i) }
-      end
+    passthrough.merge(available_translations: available_translations)
+  end
+
+  def passthrough
+    item.links.each_with_object({}) do |(link_type, passthrough_hash), result|
+      passthrough, ids = passthrough_hash.partition { |type| type.is_a?(Hash) }
+      hashes = passthrough.map { |attributes| present(ContentItem.new(attributes)) }
+      content_items = ids.map { |id| present(ContentItem.find_by(content_id: id)) }
```

Comment:
Yeah, very unlikely in this case, but also no need to be that aggressive 👍 
I ended up taking the old query, since it did a bit more filtering (also unnecessary) but good to keep a reference too until this code is completely removed.